web_src/js/features/comp/ComboMarkdownEditor.js

```diff
@@ -204,11 +203,87 @@ class ComboMarkdownEditor {
     });
   }
 
-  prepareEasyMDEToolbarActions() {
+  prepareEasyMDEToolbarActions(EasyMDE) {
     this.easyMDEToolbarDefault = [
```

Comment:
this is only the "default". "repo-wiki.js" also has its own toolbar.

Reply:
Hmm yes, wiki still broken

<img width="826" alt="image" src="https://user-images.githubusercontent.com/115237/235745907-cb4b83ec-1462-4c26-9126-bd582f7a706d.png">

Reply:
I guess it could keep the string name, and replace them with Gitea's definition (just like gitea-switch-to-textarea or gitea-checkbox-checked)


( Oops, sorry for wrong comment 😭 )

Reply:
Wiki fixes, idk why preview icon does not show:

<img width="659" alt="Screenshot 2023-05-02 at 20 10 13" src="https://user-images.githubusercontent.com/115237/235749728-da2490f2-d06e-41b6-ba61-3812d5cfc343.png">

Reply:
Preview icon only shows in fullscreen:

<img width="167" alt="image" src="https://user-images.githubusercontent.com/115237/235750608-e250838b-39de-4851-ab3d-edf397018ac6.png">